src/NUnitEngine/mock-assembly/mock-assembly.csproj

```diff
@@ -1,12 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>NUnit.Tests</RootNamespace>
-    <TargetFrameworks>net20;netcoreapp1.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net35;netcoreapp1.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="NUnit" Version="3.11.0" />
+  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp1.1'">
+    <PackageReference Include="NUnit" Version="3.12.0" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'!='netcoreapp1.1'">
+    <PackageReference Include="NUnit" Version="3.13.0-dev-06899" />
```

Comment:
Why is it that you don't use `3.13.0-dev-06899` for all target frameworks?

Reply:
The latest master build of the framework has dropped support for .NET Standard 1.x. The Engine currently still has a .NET Standard 1.x build, but in my mind this is one more good reason to remove that. There's an issue filed for that.